### DISCUSSION TITLE
How to get the phase of each path received by TX?

### DISCUSSION THREAD

**Discussion by vcyycv0** [NVlabs/sionna#458] [General]

**Comment by SebastianCa** [comment]
Hi @vcyycv0,

you can get the phase of paths from the channel impulse response. Please see the [Sionna Ray Tracing Tutorial Notebook](https://nvlabs.github.io/sionna/examples/Sionna_Ray_Tracing_Introduction.html#From-Paths-to-Channel-Impulse-Responses) for further details.

I hope this helps.

**Reply by SebastianCa** [reply]
As this is not an issue with the Sionna code, I've converted the thread to a discussion.

**Comment by vcyycv0** [comment]
thanks for your answer. I solved this problem.
